src/components/Main.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+import Header from './Header'
+class Main extends Component {
```

Comment:
If you dont have any logic inside of component, you can use functional components

Reply:
Functional Component Updated